src/ui/server/helpers/mappings/map-name-fields/index.test.ts

```diff
@@ -16,14 +15,6 @@ const mockStringWithCharacterCodes = String.raw`mock${characterCodes}`;
 
 const mockApplicationWithCharacterCodes = {
   ...mockApplication,
-  business: {
-    ...mockApplication.business,
```

Comment:
Should we not keep `business`, and just remove `businessContactDetail`?

Reply:
So the only reason business was there was for businessContactDetail.  It is only mapping buyer fields now